### ISSUE TITLE
Minimum level for Microsoft.AspNetCore in Request logging doesn't work

### ISSUE THREAD

**Issue by avengerstark** [serilog/serilog-aspnetcore#142]
Hi there!
When I set minimum level  for Microsoft.AspNetCore  to "Warning", I see in logs level "Information".
My config:
```json
  "Serilog": {
    "MinimumLevel": {
      "Default": "Information",
      "Override": {
        "System": "Warning",
        "Microsoft": "Warning",
        "Microsoft.AspNetCore": "Warning"
      }
    }
```

**Comment by nblumhardt** [comment]
Hi @avengerstark - can you please add your C# Serilog configuration (i.e. using `LoggerConfiguration`), and some examples of the events that are slipping through? Thanks!

**Comment by avengerstark** [comment]
Hi
Serilog configuration:
```c#
Log.Logger = new LoggerConfiguration()
                .ReadFrom.Configuration(Configuration)
                .CreateLogger();
```
Startup.cs ->Configure:
```c#
public void Configure(IApplicationBuilder app, IHostingEnvironment env)
{
     // ...
     app.UseSerilogRequestLogging();
     //...

}
```
Logs:
2019-10-13 13:21:41.650 +06:00 [Information] HTTP "POST" "/MeteringDevice/GetList" responded 200 in 112.0951 ms
2019-10-13 13:21:41.729 +06:00 [Information] HTTP "GET" "/MeteringDevice/GetCoefficientConversions" responded 200 in 1.2049 ms
2019-10-13 13:21:41.882 +06:00 [Information] HTTP "GET" "/MeteringDevice/GetCoefficientConversions" responded 200 in 4.5475 ms

**Comment by alexvaluyskiy** [comment]
@avengerstark It is a known issue, and will be fixed in 3.1.0. See #123

**Comment by nblumhardt** [comment]
:+1: 

Although in the case of this event, the namespace will be `"Serilog"` rather than `"Microsoft"`.

3.1.0 is on its way through CI now :-)
